ajax/libs/mrnode/1.0.2/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "mrnode",
+  "version": "1.0.2",
+  "description": "A Javascript Library for Node.js Functions",
+  "main": "dist/js/main.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/talonbragg/Mr.Node.git"
+  },
+  "keywords": [
+    "mrnode",
+    "Mr.Node",
+    "node"
+  ],
+  "author": "Talon Bragg <pvpgamecrash@gmail.com> (http://talonbragg.com)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/talonbragg/Mr.Node/issues"
+  },
+  "homepage": "https://www.mrnode.tk",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/talonbragg/Mr.Node.git",
+    "basePath": "dist/js",
+    "files": [
+      "main.min.js",
+      "main.js"

```

Comment:
Let's use `"**/*"` instead of these two. It will fetch all files under `dist` directory.